src/xr_3da/entry_point.cpp

```diff
@@ -79,13 +79,13 @@ int main(int argc, char *argv[])
 
     try
     {
-        char* commandLine = "";
+        char* commandLine = nullptr;
         int i;
         if(argc > 1)
         {
             size_t sum = 0;
             for(i = 1; i < argc; ++i)
-                sum += strlen(argv[i]) + 1;
+                sum += strlen(argv[i]) + 2;
```

Comment:
Get rid of magic value "2", please

Reply:
this is size of delimiter (space) + terminating NULL. strlen(" \0") is less magic?

Reply:
sum += strlen(argv[i]) + sizeof(DELIMITER) + sizeof ('\0') looks more clear than '2', isn't it?

Reply:
We can add a comment. Comments are acceptable in code for situations like this. And, please, remember about docs. So I think an explanatory comment - is the must.

Reply:
I agree with you, the comment could help us in this case too. However I like it better when the code itself is "self-commented".

Reply:
Yes, I agree with you too. "Self-commented" code - it's smth like a good practice. And this is really good. Bug again, in some cases.. Magic numbers, right? So yes, I'd prefer a simple comment like:
```// strlen("\0") = 2```
Smth like this, you know what I mean